packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,226 @@
+</template>
+<script>
+import noUiSlider from "nouislider";
+import wNumb from "wnumb";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    slidersInitialValues: {
+      type: Array,
+      default: () => [0, 1],
+    },
+    /*
+     * Minimal value of the slider
+     */
+    min: {
+      type: Number,
+      default: 0,
+    },
+    /*
+     * Maximum value of the slider
+     */
+    max: {
+      type: Number,
+      default: 0,
+    },
+    /*
+     * The value of moving the slider on the scale
+     */
+    step: {
+      type: Number,
+      default: 1,
+    },
+    /*
+     * Disabling the slider
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Horizontal orientation if true, vertical orientation if false
+     */
+    horizontalOrientation: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Ltr orientation if true, rtl orientation if false
+     */
+    ltrDirection: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Boolean for showing the tooltips
+     */
+    tooltips: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Config for toolips formatting (from wNumb library):
+     * decimals - number of digits after dot/comma
+     * mark - sign separating the fraction value from integer
+     * thousand - sign separating the thousands from hundreds
+     * prefix - sign before value
+     * suffix - sign after the value
+     * negative - sign for negative values, by default it is set to "-"
+     */
+    formatTooltipsValues: {
+      type: Object,
+      default: () => {
+        return {
+          decimals: 2,
+          mark: ".",
+          thousand: " ",
+          prefix: "$",
+          suffix: "",
+          negative: "",
+        };
+      },
+    },
+  },
+  data() {
+    return {
+      config: {},
+    };
+  },
+  watch: {
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
+          return this.updateRangeOptions({
+            range: {
+              min: value >= this.max ? this.max - this.step : value,
+              max: this.max,
+            },
+          });
+        }
+      },
+      immediate: true,
+    },
+    max: {
+      handler(value) {
+        return this.updateRangeOptions({
+          range: {
+            min: this.min,
+            max: value <= this.min ? this.min + this.step : value,
+          },
+        });
+      },
+      immediate: true,
+    },
+    step: {
+      handler(value) {
+        return this.updateRangeOptions({
+          step: value,
+        });
+      },
+      immediate: true,
+    },
+    tooltips: {
+      handler(value) {
+        if (value) {
+          value =
+            this.slidersInitialValues.length === 2
+              ? [
+                  wNumb(this.formatTooltipsValues),
+                  wNumb(this.formatTooltipsValues),
```

Comment:
Twice same?
Maybe it would be worth to set some const:
```ts
const wnumbed = wNumb(this.formatTooltipsValues);

// and then
[ wnumbed, wnumbed ]
// and
[ wnumbed ]
```

Reply:
it looks better, thanks :)